src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * Updated OTel SDK package version to 1.3.0
   ([#411](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/411))
+* Add GC heap size and count in Runtime metrics
```

Comment:
gc count was already there. this is a breaking change (which is okay), so need to describe this a bit more clearly.

Reply:
Reworded it as "Add GC heap size and refactor GC count as multi-dimensional metrics in Runtime metrics"